app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -614,11 +614,11 @@
 				TargetAttributes = {
 					510254221C0A60670045FCD8 = {
 						CreatedOnToolsVersion = 7.1;
-						DevelopmentTeam = 4R4XY9R8JR;
+						DevelopmentTeam = GN257VN844;
```

Comment:
This will mean I will be unable to do releases so you'll have to that that over @orta